conf/base.config

```
@@ -159,4 +159,14 @@ process {
         cpus   = { check_max (8     * task.attempt, 'cpus'   ) }
         memory = { check_max (20.GB * task.attempt, 'memory' ) }
     }
+
+    withName: MAXBIN2 {
+        // often fails when insufficient information, so we allow it to gracefully fail without filling the pipeline
+        errorStrategy = { task.exitStatus in [ 1, 255 ] ? 'ignore' : 'retry' }
+    }
+
+    withName: DASTOOL_DASTOOL {
+        // if SCGs not found, bins cannot be assigned and DAS_tool will die
```

```suggestion
        // if SCGs not found, bins cannot be assigned and DAS_tool will die with exit status 1
```
correct? just to add a justification for the line below.